proto-sync.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+
+# This script synchronyzes exonum protobuf specification files with the `exonum-proto-sources` repository, where they
+# could be used from by various Exonum clients.
+#
+# It is supposed that repo with proto files has the same branches as exonum repo. Thus, this allows to have different
+# sets of proto files for certain branch. Also, script updates proto files for current branch only. For example, in
+# order to sync files for exonum branch "feature_X", one should execute this script from the local branch "feature_X".
```

Comment:
It is supposed that exonum-proto-sources repository has the same branches as Exonum one. The script does not perform checkout, so it must be executed from the local branch with the name equal to the name of the target Exonum repository branch.

Reply:
Exactly. This is the intended behavior that we ended up with after discussion. However, it's still not clear to me how to support different sets of proto files for different branches. I guess, for the moment the best approach would be to have different sets hardcoded for different branches. Current code reflects the `dynamic_services` branch.

Reply:
It was actually a suggestion to rephrase this paragraph :wink: